allennlp/predictors/bidaf.py

```diff
@@ -1,8 +1,14 @@
+# pylint: disable=protected-access
+from copy import deepcopy
+from typing import Dict, List
 from overrides import overrides
-
+import numpy as np
 from allennlp.common.util import JsonDict
 from allennlp.data import Instance
+from allennlp.data.tokenizers import Token
 from allennlp.predictors.predictor import Predictor
+from allennlp.data.fields import IndexField, TextField, ListField, \
```

Comment:
I don't have a strong opinion about this particular use of backslash, though I think @joelgrus would recommend using parens to accomplish the continuation instead.